tests/product/test_server_install.py

```diff
@@ -407,111 +389,6 @@ def test_install_interactive_with_hostnames(self):
             self.assert_has_default_connector(container)
             self.assert_has_jmx_connector(container)
 
-    def test_install_interactive_with_ips(self):
```

Comment:
This can be removed because we're not really testing anything here, just Fabric's ability to connect to a host using an IP.